Templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+Concise description of what was done
+
+## Solve (Issues)
```

Comment:
I think this line should be (fixes #issue_number) and this should be at the end of pull request title